test/int/fixtures/testUsing.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { IFixture } from './fixture';
+import { PromiseOrNot } from 'vscode-chrome-debug-core';
+
+/** Run a test doing the setup/cleanup indicated by the provided fixtures */
+async function testUsingFunction<T extends IFixture>(
+    expectation: string,
+    fixtureProvider: () => PromiseOrNot<T>,
```

Comment:
Let's pass the test expectation as a param to the fixtureProvider so that we don't have to declare it twice

Reply:
Let's pass the test expectation as a param to the fixtureProvider so that we don't have to declare it twice

Reply:
Passing a parameter to fixtureProvider is a great idea. I'll pass the context so we can get the title from there.